### ISSUE TITLE
Insufficient contrast on light gray vertical bars on breadcrumbs

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/onrr.gov-site#2770]
![image.png](https://images.zenhubusercontent.com/5c59d3cd7592186396262002/ea00f8b1-8e2b-4081-b3d1-4c4e881c5d07)

**Comment by eelzi-ONRR** [comment]
Hex for the is currently #9e9e9e. Change to #767676

**Comment by greensomes** [comment]
@lpgoldstein , this is ready for review in preview.

**Comment by lpgoldstein** [comment]
@greensomes looks good

**Comment by greensomes** [comment]
This has been deployed to dev and will go live with the ONRR Editor release.

**Comment by lpgoldstein** [comment]
looks good in dev
